PhysicsTools/NanoAOD/python/custom_jme_cff.py

```diff
@@ -25,7 +25,8 @@
   'pfDeepFlavourJetTags:probb','pfDeepFlavourJetTags:probbb','pfDeepFlavourJetTags:problepb',
```

Comment:
pfCombinedMVAV2BJetTags and pfCombinedInclusiveSecondaryVertexV2BJetTags are already in the central nano, why are they added also in the JMEnano with a different name ?

    btagCMVA = Var("bDiscriminator('pfCombinedMVAV2BJetTags')",float,doc="CMVA V2 btag discriminator",precision=10),
    btagCSVV2 = Var("bDiscriminator('pfCombinedInclusiveSecondaryVertexV2BJetTags')",float,doc=" pfCombinedInclusiveSecondaryVertexV2 b-tag discriminator (aka CSVV2)",precision=10)

Reply:
@camclean @alefisico any comment ?

Reply:
These variables are included in the central nano for the `Jet` collection (AK4 CHS) but in our JMEnano we included other AK4 collections (PUPPI/PF) where we want to have the same content for comparison. So we need to include these variables in the new jet collections.